Update cht-sync image (#1292)
medic/cht-docs: content/en/core/overview/data-flows-for-analytics/cht-sync.md

#### Superset

> We recommend [Apache Superset](https://superset.apache.org/) as the Data Visualization Tool, it is a free an open source platform for creating data dashboards.

Reviewer: @Phil-Mwago what was omitted , was it not Superset. Kindly clarify.

Author: @esthermmoturi yes, superset will not be part of the bundled containers in the cht-sync config, however we will recommend it as the visualization tool since its open source and a provide a script to connect.

> We recommend [Apache Superset](https://superset.apache.org/) as the Data Visualization Tool, it is a free an open source platform for creating data dashboards.

Reviewer 2: ```suggestion
We recommend [Apache Superset](https://superset.apache.org/) as the Data Visualization Tool. Superset is a free, open-source platform for data exploration and data visualization.
```

Reviewer 2: I reformulated a bit of the sentence above as I find `data dashboards` unclear.
